student.rb

```diff
@@ -12,3 +18,4 @@ def play_hooky
     '¯\(ツ)/¯'
   end
 end
+f
```

Comment:
- [ ] Please consider removing meaningless pieces of code, they make your codebase look ugly and hard to read! 🤓 🔧